### DISCUSSION TITLE
Css background-color attribute

### DISCUSSION THREAD

**Discussion by m-bakoulis** [givanz/Vvveb#239] [Q&A] [answered]
Hi givanz, does the editor has the option to setup the background-color attribute on a section or div? I can easily find the background-image option but I am not able to find how to set a color for the background.

**Comment by givanz** [comment, marked as answer]
Hi

Background color can be changed from style tab > display > background color

Section elements didn't have options in style tab, I changed this in `latest.zip` update and now background color also works for section element.

Changed file `/public/js/vvvebjs/section.js`

![section-bg-color](https://github.com/user-attachments/assets/42b207b3-f107-4506-9bf8-2f730f6bcc6f)

**Reply by m-bakoulis** [reply]
Thank you for this new feature givanz!
